proto/regen/group/v1alpha1/types.proto

```diff
@@ -0,0 +1,258 @@
+syntax = "proto3";
+
+package regen.group.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/group";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/any.proto";
+
+// Member represents a group member with an account address,
+// non-zero weight and metadata.
+message Member {
+
+    // address is the member's account address.
+    string address = 1;
+    
+    // weight is the member's voting weight that should be greater than 0.
+    string weight = 2;
+    
+    // metadata is any arbitrary metadata to attached to the member.
+    bytes metadata = 3;
+}
+
+// ThresholdDecisionPolicy implements the DecisionPolicy interface
+message ThresholdDecisionPolicy {
+    option (cosmos_proto.implements_interface) = "DecisionPolicy";
+
+    // threshold is a fix weight value that must be met or exceeded for a proposal to succeed.
```

Comment:
does this refer to number of votes or total weight? may be being explicit here would be handy

Reply:
@anilCSE what do you think about this? 
```suggestion
    // threshold is the minimum weighted sum of yes votes that must be met or exceeded for a proposal to succeed.
```

Reply:
Looks good.